stdlib/jdbc/src/main/java/org/ballerinax/jdbc/datasource/SQLDatasource.java

```diff
@@ -119,6 +123,10 @@ public boolean isPoolShutdown() {
         return poolShutdown;
     }
 
+    public boolean isKeyRetrievalSupported() {
+        return this.supportsGetGeneratedKeys;
```

Comment:
Normal practice is boolean returned function name = `is` + `boolean` variable name. Can we rename the method name or variable name to follow the same practice?